internal/backend/remote/backend_common.go

```diff
@@ -419,7 +419,8 @@ func (b *Remote) checkPolicy(stopCtx, cancelCtx context.Context, op *backend.Ope
 		case tfe.PolicyHardFailed:
 			return fmt.Errorf(msgPrefix + " hard failed.")
 		case tfe.PolicySoftFailed:
-			runUrl := fmt.Sprintf(runHeader, b.hostname, b.organization, op.Workspace, r.ID)
+			runUrl := b.Colorize().Color(strings.TrimSpace(fmt.Sprintf(runHeader, b.hostname, b.organization, op.Workspace, r.ID)) + "\n")
+			b.CLI.Output(runUrl)
```

Comment:
Why was this caller added? Was it missing and that was part of the bug? It looks like runUrl was previously being returned as part of an error message.

As an aside, "runUrl" should probably be formatted "runURL" because we typically capitalize acronyms in go

Reply:
Right now in main, [this line](https://github.com/hashicorp/terraform/blob/8da1c006f9d7f381e644a5f0e2c25094ea29571d/internal/cloud/backend_common.go#L394) does not colorize `runHeader` which outputs to the CLI, so I am calling here it here.

`runUrl` is how it appears in main, but I will update all of them to `runURL` 👍

Reply:
Or are you asking why I made this change in `internal/backend/remote/`, specifically? I made the change in both `remote` and `cloud`.

Reply:
Or are you only referring to line 423 `b.CLI.Output(runUrl)` in this file?

Reply:
@brandonc nvm! fixed!